framework/test/utils/node/state_store_mock.ts

```diff
@@ -18,7 +18,7 @@ import { createFakeDefaultAccount } from './account';
 import { AccountStateStore, Account, AccountDefaultProps } from '../../../src/modules/base_asset';
 
 export interface ChainState {
-	readonly lastBlockHeaders: BlockHeader[];
+	lastBlockHeaders: BlockHeader[];
```

Comment:
this shouldn't be modified

Reply:
I can add a comment there for reference, there are a lot of use cases in tests to change the last block headers, if we make it readonly, make it redundant to re-initialize the whole StateStoreMock object.